app/utils/prepareAnswer.js

```diff
@@ -82,6 +93,24 @@ function prepareStatsDistrictAnswer(body, stateName) {
     return data
 }
 
+function prepareStateTestStat(body, stateName) {
+
+    let stateWise = body['states_tested_data'], stateDataIndex = -1
+    for (let index = 0; index < stateWise.length; index++) {
+        let currentData = stateWise[index]
+        if (currentData['state'].toLowerCase() === stateName.toLowerCase())
```

Comment:
You can `break;` this `for` loop as soon as we find the last instance of our data of the requested state instead of running the loop for all the items.

Reply:
aaye nice, will do this too xD

Reply:
break added in for loop